Chucker 3.3.0 is incompatible with projects using Kotlin 1.4.20 and @Parcelize

## :writing_hand: Describe the bug
`@Parcelize` has moved out of the Kotlin Android Extensions (which is deprecated) into its own compiler plugin. It's using a different package name, but the new runtime still defines the old classes to provide deprecation errors. With both the old and new runtimes defining the same classes, this causes Android's `checkDebugDuplicateClasses` build task to fail.

## :bomb: Steps to reproduce
<!-- How we can reproduce the behavior: -->
1. Create Android application using Kotlin 1.4.20-M2 (or whatever version is now available) and using `plugins { kotlin("plugin.parcelize") }`.
2. Add Chucker 3.3.0.
3. Try to build.
4. ```
    > Task :app:checkDebugDuplicateClasses FAILED

    FAILURE: Build failed with an exception.

    * What went wrong:
    Execution failed for task ':app:checkDebugDuplicateClasses'.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
       > Duplicate class kotlinx.android.parcel.IgnoredOnParcel found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parceler found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parceler$DefaultImpls found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parcelize found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.RawValue found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.TypeParceler found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.WriteWith found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)

         Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
    ```


## :wrench: Expected behavior
Success

## :camera: Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

## :iphone: Tech info
 - Device: <!-- e.g. Nexus 6P -->
 - OS: <!-- e.g. 7.1.1 -->
 - Chucker version: <!-- e.g. 3.1.2 -->

## :page_facing_up: Additional context
<!-- Add any other context about the problem here. -->

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,10 +8,12 @@ Please add your entries according to this format.
 ### Changed
 
 * Bumped `targetSDK` and `compileSDK` to 30 (Android 11).
+* Removed `kotlin-android-extensions` plugin.
 
 ### Fixed
 
 * Fixed memory leak in MainActivity [#465].
+* Fixed build failure for projects with new `kotlin-parcelize` plugin [#480].
 
 ## Version 3.3.0 *(2020-09-30)*
 
@@ -450,3 +452,4 @@ Initial release.
 [#410]: https://github.com/ChuckerTeam/chucker/issues/410
 [#422]: https://github.com/ChuckerTeam/chucker/issues/422
 [#465]: https://github.com/ChuckerTeam/chucker/issues/465
+[#480]: https://github.com/ChuckerTeam/chucker/issues/480

--- a/library/build.gradle
+++ b/library/build.gradle
@@ -1,6 +1,5 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
 android {

--- a/sample/build.gradle
+++ b/sample/build.gradle
@@ -1,6 +1,5 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
@@ -11,6 +10,10 @@ android {
         applicationId "com.chuckerteam.chucker.sample"
         versionCode 1
         versionName "1.0"
+    }
+
+    buildFeatures {
+        viewBinding = true
     }
 
     buildTypes {

--- a/sample/src/main/java/com/chuckerteam/chucker/sample/MainActivity.kt
+++ b/sample/src/main/java/com/chuckerteam/chucker/sample/MainActivity.kt
@@ -4,9 +4,11 @@ import android.os.Bundle
 import android.view.View
 import androidx.appcompat.app.AppCompatActivity
 import com.chuckerteam.chucker.api.Chucker
-import kotlinx.android.synthetic.main.activity_main_sample.*
+import com.chuckerteam.chucker.sample.databinding.ActivityMainSampleBinding
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var mainBinding: ActivityMainSampleBinding
 
     private val client: HttpBinClient by lazy {
         HttpBinClient(applicationContext)
@@ -14,14 +16,16 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main_sample)
 
-        do_http.setOnClickListener { client.doHttpActivity() }
-        trigger_exception.setOnClickListener { client.recordException() }
+        mainBinding = ActivityMainSampleBinding.inflate(layoutInflater)
 
-        with(launch_chucker_directly) {
-            visibility = if (Chucker.isOp) View.VISIBLE else View.GONE
-            setOnClickListener { launchChuckerDirectly() }
+        with(mainBinding) {
+            setContentView(root)
+            doHttp.setOnClickListener { client.doHttpActivity() }
+            triggerException.setOnClickListener { client.recordException() }
+
+            launchChuckerDirectly.visibility = if (Chucker.isOp) View.VISIBLE else View.GONE
+            launchChuckerDirectly.setOnClickListener { launchChuckerDirectly() }
         }
 
         client.initializeCrashHandler()

